py3status/modules/xrandr_rotate.py

```diff
@@ -134,8 +123,22 @@ def xrandr_rotate(self):
         elif self.displayed == self._get_current_rotation_icon(all_outputs):
             response['color'] = self.py3.COLOR_GOOD
 
+        self.scrolling = False
         return response
 
+    def on_click(self, event):
+        """
+        Click events
+            - left click & scroll up/down: switch between rotations
+            - right click: apply selected rotation
+        """
+        button = event['button']
+        if button in [1, 4, 5]:
+            self.scrolling = True
+            self._switch_selection()
+        elif button == 3:
+            self._apply()
+
```

Comment:
Why has this function moved?

Reply:
Overall consistency. Most of the modules have `on_click` on bottom.

Reply:
That is fine but it should be in it's own PR

Reply:
Do you want me to do that? `y/n`

Reply:
You can leave it this time, but in future please keep thing separate.

It makes the whole merge process much much slower and as you'll know currently this is an issue on the project.

Hopefully it will improve soon.